plugins/woocommerce-admin/client/marketplace/components/category-selector/category-dropdown.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { Dropdown } from '@wordpress/components';
+import { chevronDown, chevronUp, Icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { Category } from './category-selector';
+
+function DropdownContent( props: {
+	readonly categories: Category[];
+} ): JSX.Element {
+	return (
+		<ul className="woocommerce-marketplace__category-dropdown-list">
+			{ props.categories.map( ( category ) => (
+				<li
+					className="woocommerce-marketplace__category-dropdown-item"
+					key={ category.slug }
+				>
+					<button className="woocommerce-marketplace__category-dropdown-item-button">
+						{ category.label }
+					</button>
+				</li>
+			) ) }
+		</ul>
+	);
+}
+
+type CategoryDropdownProps = {
+	label: string;
+	categories: Category[];
+	className?: string;
+	buttonClassName?: string;
+	contentClassName?: string;
+	arrowIconSize?: number;
+};
+
+export default function CategoryDropdown(
+	props: CategoryDropdownProps
+): JSX.Element {
+	return (
+		<Dropdown
+			renderToggle={ ( { isOpen, onToggle } ) => (
+				<button
+					onClick={ onToggle }
+					className={ props.buttonClassName }
+					aria-label="Toggle category dropdown"
```

Comment:
```suggestion
					aria-label={ __( 'Toggle category dropdown', 'woocommerce' ) }
```

Reply:
Thank you! I missed this by accepting the Github Copilot suggestion.